constants/evm.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const (
+	EvmCompatibleAddressPattern = "^0x[0-9a-fA-F]{40}$"
```

Comment:
let's make `0x` optional in case someone provides an address without `0x` -> HexToAddress will add it for himself.